site/gatsby-site/server/fields/common.ts

```diff
@@ -209,26 +180,37 @@ export const sendEmail = async ({ recipients, subject, dynamicData, templateId }
     }
 
     try {
-        const emailData = buildEmailData(recipients, subject, dynamicData, emailTemplateBody);
 
-        const mailersend = new MailerSend({
-            apiKey: config.MAILERSEND_API_KEY,
-        });
+        const bulk: EmailParams[] = [];
 
-        const personalizations = emailData.personalizations
-            .map((personalization) => ({ data: personalization.substitutions, email: personalization.to.email }));
+        for (const recipient of recipients) {
+
+            const personalizations = [{
+                email: recipient.email,
+                data: {
+                    ...dynamicData,
+                    email: recipient.email,
+                    userId: recipient.userId,
+                    siteUrl: config.SITE_URL,
+                }
+            }]
 
-        const emailParams = new EmailParams()
-            .setFrom({ email: config.NOTIFICATIONS_SENDER, name: config.NOTIFICATIONS_SENDER_NAME })
-            .setTo(recipients.map((recipient) => new Recipient(recipient.email)))
-            .setPersonalization(personalizations)
-            .setSubject(subject)
             // We have to do this because MailerSend is escaping the placeholders containing html tags
-            .setHtml(replacePlaceholdersWithAllowedKeys(emailData.html, dynamicData, ['developers', 'deployers', 'entitiesHarmed']));
-        //TODO: add a text version of the email
-        // .setText("Greetings from the team, you got this message through MailerSend.");
+            const html = replacePlaceholdersWithAllowedKeys(emailTemplateBody, dynamicData, ['developers', 'deployers', 'entitiesHarmed'])
+
+            const emailParams = new EmailParams()
+                .setFrom({ email: config.NOTIFICATIONS_SENDER, name: config.NOTIFICATIONS_SENDER_NAME })
+                .setTo([new Recipient(recipient.email, '')])
```

Comment:
```suggestion
                .setTo([new Recipient(recipient.email, /* recipient.name */ '')])
```

or something equivalent / nice to read.
(lint: magic nil strings)